bundles/mapping/mapmodule/oskariStyle/filter.js

```diff
@@ -113,3 +107,20 @@ const filterProperty = (property, featValue) => {
     Oskari.log('Oskari.mapping.mapmodule.oskariStyle').debug('Optional style has redundant keys: ' + ignoredOperators);
     return false;
 };
+
+const likeMatcher = (featValue, like) => {
+    const endsWildCard = like.charAt(like.length - 1) === WILD_CARD;
+    const startsWildCard = like.charAt(0) === WILD_CARD;
+    let val;
+    if (endsWildCard && startsWildCard) {
+        val = like.substr(1, like.length - 2);
+    } else if (endsWildCard) {
+        val = like.substr(0, like.length - 2);
+    } else if (startsWildCard) {
+        val = like.substr(1);
+    } else {
+        Oskari.log('Oskari.mapping.mapmodule.oskariStyle').info('Optional style\'s filter like operator does not have wild card, use value operator instead');
+        return false;
```

Comment:
Should we still compare value instead of always returning false if there is no wildcard?
```suggestion
        return featValue == like;
```

Reply:
"Working as intended"